assets/src/application/services/theme/ThemeProvider.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ThemeProvider as ChakraThemeProvider } from '@chakra-ui/core';
+
+import theme from './theme';
+
+const ThemeProvider: React.FC = ({ children }) => {
+	return <ChakraThemeProvider theme={theme}>{children}</ChakraThemeProvider>;
+};
+
+export default ThemeProvider;
```

Comment:
Since we don't create our own context, I didn't place it inside `/application/services/context`